libs/cart/src/facades/cart/cart-facade.interface.ts

```diff
@@ -48,7 +48,8 @@ export interface DaffCartFacadeInterface<
   orderResultLoading$: Observable<boolean>;
 	orderResultErrors$: Observable<string[]>;
 	orderResult$: Observable<V>;
-	orderResultId$: Observable<V['id']>;
+	orderResultId$: Observable<V['orderId']>;
+	orderResultCartId$: Observable<V['cartId']>;
```

Comment:
I don't think orderResultCartId should be on the facade. The app dev should never need it. If he/she does, it can be accessed with the selector.

Reply:
The app dev would need the cart ID to fetch guest orders. I have a PR out for that functionality.

Reply:
Oh, I'd thought that was being handled internally by daffodil, but I guess it makes sense they would need to pass it.